model/src/cfd.rs

```diff
@@ -704,16 +698,19 @@ impl Cfd {
         &self,
         now: OffsetDateTime,
     ) -> Result<(Txid, BitMexPriceEventId), CannotRollover> {
-        let expiry_timestamp = self.expiry_timestamp().ok_or(CannotRollover::NoDlc)?;
+        self.can_rollover()?;
+
+        let dlc = match &self.dlc {
```

Comment:
❓Any particular reason why you prefer the `match` over the `ok_or`? I think it should be equivalent.

Reply:
Nope, I just forgot!